### PR TITLE
feat(pylint): Add support for pylint version 1.6.4

### PR DESCRIPTION
**Pull request by agb80**
Make changes needed for support verison 1.6.4 of pylint.

According with our conversation on #58 in this PR I'm proposing the changes needed for support latest pylint version. Also get back pylint20 on tox to point to master as this commit fix that build too.

### CODE REVIEW

**Comment by agb80** [comment]
PR on pylint20 failed because of this bug that @moylop260 reported: https://github.com/PyCQA/astroid/issues/358

Works good on my local machine with a virtualenv and up to date setuptools

**Comment by moylop260** [comment]
Thanks
FYI I have created a fix https://github.com/PyCQA/astroid/pull/360

Could you check if you can propose it for astroid/pylint project and version that you are using now?

File: tox.ini [hunk 1]
```diff
@@ -14,9 +14,9 @@ deps =
 [testenv]
 deps =
     py27-pylint14: astroid==1.4.6
-    py27-pylint14: pylint==1.5.6
-    py27-pylint20: git+https://github.com/PyCQA/astroid@461e016
-    py27-pylint20: git+https://github.com/PyCQA/pylint@0495355
+    py27-pylint14: pylint==1.6.4
+    py27-pylint20: git+https://github.com/PyCQA/astroid@master
```
Comment (moylop260) [inline tox.ini]: Please freeze the 2.0 version in order to avoid future builds red.

Comment (moylop260) [inline tox.ini]: FYI I'm using a good newest sha without errors in our fork:
https://github.com/Vauxoo/pylint-odoo/blob/master/tox.ini#L25

Comment (moylop260) [inline tox.ini]: Use latest astroid stable version 1.4.7

**Comment by agb80** [comment]
@moylop260  please help; I'm not very familiar with travis and the way it build the test environments. 

I already tried using the latest commits on astroid & pylint and locally builds but travis failed; tried with the commits you have on your fork travis failed again, and tried updating just astroid for include the patch you done while keeping master on your pinned commit and failed again. All times my local dev & test machines running tox -e py27-pylint20,profile-stats build without problem.

What do you think could be the problem and how can I solved it?

**Comment by moylop260** [comment]
My secret weapon to reproduce travis error on local is t2d with the original image of travis
Try it: `travisfile2dockerfile --docker-image=quay.io/travisci/travis-python --docker-user=travis git@github.com:OCA/pylint-odoo.git pull/67`
screen shot 2016-09-05 at 9 09 32 pm

**Comment by moylop260** [comment]
Hello?

**Comment by agb80** [comment]
I'm really sorry but for good and for bad we are plenty of work at this time. I'm get back to this as soon as I can or fell free to finish for me if you wish.

**Comment by moylop260** [comment]
Superseeded by https://github.com/OCA/pylint-odoo/pull/69

**Pull request closed without merging**